.github/workflows/master.yml

```diff
@@ -261,23 +261,17 @@ jobs:
         run: |
           make dist
         env:
-          PULUMI_NODE_MODULES: ${{ runner.temp }}/opt/pulumi/node_modules
```

Comment:
Is there any reason why we need to remove this and not update it?

Reply:
Because we install our node_modules where we build them and then link. We don't put them in an external directory. `PULUMI_NODE_MODULES` is no longer used.